ticdc/ticdc-faq.md

```diff
@@ -78,7 +78,13 @@ If a replication task starts after the TiCDC service starts, the TiCDC owner upd
 
 If the replication task is suspended longer than the time specified by `gc-ttl`, the replication task enters the `failed` status and cannot be resumed. The PD corresponding service GC safepoint will continue.
 
-The Time-To-Live (TTL) that TiCDC sets for a service GC safepoint is 24 hours, which means that the GC mechanism does not delete any data if the TiCDC service can be recovered within 24 hours after it is interrupted.
+The default Time-To-Live (TTL) that TiCDC sets for a service GC safepoint is 24 hours, which means that the GC mechanism does not delete any data if the TiCDC service can be recovered within 24 hours after it is interrupted.
+
+## How to recover a replication task after it fails?
+
+1. Use `cdc cli changefeed query` to query the error information of the replication task and fix the error as soon as possible.
+2. Increase the value of `gc-ttl` to allow more time to fix the error, ensuring that the replication task does not enter the `failed` status due to the replication delay exceeding `gc-ttl` after the error is fixed.
+3. After evaluating the impact on the system, increase the value of [`tidb_gc_life_time`](/system-variables.md#tidb_gc_life_time-new-in-v50) in TiDB to prevent GC from retaining data, ensuring that the replication task does not enter the `failed` status due to GC cleaning data after the error is fixed.
```

Comment:
> ... to prevent GC from retaining data, ...

Retaining or deleting?

I think the original text is kind of ambiguous.

> ...以达到阻止 GC 保留数据的目的

The actual purpose is to "阻止 GC" and "保留数据" instead of "阻止 (GC 保留数据)" ...